jane/doc/extensions/_05-kinds/syntax.md

```diff
@@ -308,7 +308,7 @@ about its implementation.)
 
 ## Kind annotations on variables
 
-Type variables come in two flavors in OCaml: rigid and unifiable. Rigid type
+Type variables come in two flavors in OxCaml: rigid and unifiable. Rigid type
```

Comment:
Well, the statement is true of OCaml just as much as OxCaml. I'm happy for the change here, but I just wanted to point out that this is not new.